DependencyInjection/DoctrineExtension.php

```diff
@@ -343,6 +345,13 @@ protected function ormLoad(array $config, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('orm.xml');
 
+        // This check can be removed when support for Symfony < 4.2 is dropped
+        if (! interface_exists(ResetInterface::class)
+            || ! is_a(DoctrineType::class, ResetInterface::class)
+            || ! $container->getDefinition('form.type.entity')->hasTag('kernel.reset')) {
```

Comment:
If this works reliably (I’m not sure because I don’t know when autoconfig is applied, ping @nicolas-grekas) I don’t believe we need the other checks at all 🤔